tests/test_converter.py

```diff
@@ -1176,6 +1176,72 @@ def test_qasm_(self, recorder):
         assert recorder.queue[1].parameters == []
         assert recorder.queue[1].wires == Wires([2, 0])
 
+        for i in range(2, 6):
+            assert recorder.queue[i].name == "MidMeasureMP"
+            assert recorder.queue[i].wires == Wires([i - 2])
+
+    def test_qasm_measure(self):
```

Comment:
Can you run `black -l 100 pennylane-qiskit tests` to fix the `black` formatting errors?

Reply:
When I run this command, the only reformatting done is that the change I made to make `qasm_string` so that it is multiline gets reverted back into a single-line definition like it was before. Should I keep it single-line as Black is suggesting?

Reply:
Yes, please keep the formatting the black suggests for the `Formatting checks` to pass.
If you get the single-line definition, then add the strings components explicitly with `+`: 
`"abc" + "def" == "abcdef"`